manager/api/migrator/src/main/java/io/apiman/manager/api/migrator/DataMigrator.java

```diff
@@ -39,10 +40,10 @@
 @Dependent
 public class DataMigrator implements IReaderHandler {
 
+    private IApimanLogger logger = ApimanLoggerFactory.getLogger(DataMigrator.class);
```

Comment:
Any reason to not make this also static final?

Reply:
See previous